code-server/main.tf

```diff
@@ -95,6 +95,12 @@ variable "use_cached" {
   default     = false
 }
 
+variable "use_cached_extensions" {
+  type        = bool
+  description = "Uses cached copy of extensions, otherwise do a forced upgrade"
```

Comment:
Ah wait I just realized this only works if you set a manual extensions directory.  Should we say something like:

```
  description = "Uses cached copy of extensions, otherwise do a forced upgrade.  Only works if you set a custom extension directory."
```

But, perhaps we could try `code-server --list-extensions` instead?

Reply:
Awesome, i see that works in both cases.

```shell
.code-server/bin/code-server --list-extensions
dracula-theme.theme-dracula
esbenp.prettier-vscode
golang.go
hashicorp.terraform
```